Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml

```diff
@@ -20,17 +20,122 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toTopOf="@+id/submission_dispatcher_scroll"
             app:title="@{@string/submission_dispatcher_headline}" />
 
-        <include
-            android:id="@+id/submission_dispatcher_content"
-            layout="@layout/include_submission_dispatcher"
-            android:layout_width="@dimen/match_constraint"
-            android:layout_height="@dimen/match_constraint"
-            app:layout_constraintBottom_toBottomOf="parent"
+        <ScrollView
```

Comment:
I cannot fully see the last card. The scroll view should not use wrap_content

Reply:
```suggestion
          <ScrollView
            android:id="@+id/submission_dispatcher_scroll"
            android:layout_width="0dp"
            android:layout_height="0dp"
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintTop_toBottomOf="@+id/submission_dispatcher_header"
            app:layout_constraintBottom_toBottomOf="parent">
```

Reply:
Thanks @chiljamgossow